### DISCUSSION TITLE
2/3 of LDAP Users gone Missing from users list after upgrade to 0.50

### DISCUSSION THREAD

**Discussion by ips972** [open-webui/open-webui#8064] [Issues]
after upgrading docker-cuda to 0.50 from 0.48 - 2/3 of my ldap users in the system are missing, only 50 users are shown.
i had 150 users registered in the system. 

whats happening?!?

**Comment by tjbck** [comment]
Definitely very strange, we didn't change/touch any backend code regarding auth/users for 0.5.0. If I had to make an educated guess here, it's because of volume persistent issue caused by docker-end. Do keep us updated!

**Reply by ips972** [reply]
just verified by loading the system with 0.47 image using the same volumes and i can see all 125 users from LDAP registered.  now trying to figure out how to reload the docker compose config with version 0.48 back again. 
this it the command in the compose file. i didn't use a version tag so whats the full command with right tag? 
image: ghcr.io/open-webui/open-webui:cuda   <--- what do i need to add here to roll back to 0.48?

**Reply by tjbck** [reply]
@ips972 should be fixed with #8068, try pulling the latest main and let me know if the issue persists!

**Reply by ips972** [reply]
im using cuda tag and not main, for GPU support, is it fixed for cuda too?
image: ghcr.io/open-webui/open-webui:cuda

**Reply by tjbck** [reply]
Yep should be! Keep me updated

**Reply by ips972** [reply]
problem solved! new release fixed the users list. THANKS!!!
